qudipy/potential/process_nextnano.py

```
@@ -0,0 +1,418 @@
+            d = f.readlines()
+
+            xdim = [int(i) for i in d[3].split() if i.isdigit()][0]
+            ydim = [int(i) for i in d[4].split() if i.isdigit()][0]
+            zdim = [int(i) for i in d[5].split() if i.isdigit()][0]
+
+        # extract xyz coordinate data
+        with open(filename, 'r') as f:
+            d = f.readlines()
+
+            # convert list of strings to list of floats
+            data = []
+            for i in list(filter(lambda x: x != '\n', d)):
+                data.append(float(i.strip()))
+
+            x = data[:xdim]
+            y = data[xdim:xdim+ydim]
+            z = data[xdim+ydim:xdim+ydim+zdim]
+
+            return x, y, z
+
+def parse_ctrl_items(filename, ctrl_type):
+    '''
+    This function collects all of the control items in the filename string and
+    processed nextnano potential files are assumed to follow the syntax: 
+    'TYPE_C1NAME_C1VAL_C2NAME_C2VAL_..._CNNAME_CNVAL.txt'
+    where TYPE = 'Uxy' or 'Ez'. 
+    Refer to tutorial for a more explicit example.
+
+    Parameters
+    ----------
+    filename: String which is the relative path to the filename and containts 
+        control items (either name/value) information.
+       
+    Keyword Arguments
+    ----------
+    ctrl_type: Specifies what control item needs to be returned. Acceptable 
+        arguments include ['value','values','name','names'] where case is 
+        not relevant.
+
+    Returns
+    -------
+    ctrl_type: List of control items of the specific control type
+    '''
+
+    # parse string via _,\, /
+    parsed_filename = re.split(r'[_/\\]',filename)
+
+    if ctrl_type.lower() in ['value','values']:
+
+        ctrl_vals = []
+        # search through list of strings for floats from the parsed file name
+        for i in parsed_filename:
+            try:
+                if float(i) < 100:
+                    ctrl_vals.append(float(i))
+            except ValueError:
+                pass
```

What error is this catching?  It's generally bad practice to have an except: pass statement.  It seems to me that break would be better than pass.